conda/install.py

```diff
@@ -372,6 +373,10 @@ def run_script(prefix, dist, action='post-link', env_prefix=None):
     env['PKG_NAME'], env['PKG_VERSION'], env['PKG_BUILDNUM'] = \
                 str(dist).rsplit('-', 2)
     if action == 'pre-link':
+        error_and_exit("""
```

Comment:
wrap this in `sys.stderr.write()` with a `\n` at the end instead of `error_and_exit()`